src/ui/common/src/components/integrations/cards/detailCard.tsx

```diff
@@ -51,10 +53,46 @@ export const DetailIntegrationCard: React.FC<DetailIntegrationCardProps> = ({
     case 'S3':
       serviceCard = <S3Card integration={integration} />;
       break;
+    case 'Kubernetes':
+      serviceCard = <KubernetesCard integration={integration} />;
+      break;
+    case 'Lambda':
+      serviceCard = <LambdaCard integration={integration} />;
+      break;
     default:
       serviceCard = null;
   }
 
+  if (
```

Comment:
The code in this if statement is really similar to that of the section below it. 

From my understanding, all we want to do here is render the "Created on" text if the integration is NOT kubernetes or lambda, correct?

Instead, we can simplify our code and remove the 66-95 and replace  with:

```
let createdOnText null; //createdOn text component
if (integration.service !== 'Kubernetes || integration.service !== 'Lambda') {
  createdOnText = 
    (<Typography variant="body1">
      <strong>Created On: </strong>
      {new Date(integration.createdAt * 1000).toLocaleString()}
    </Typography>)
}


  return (
    <Box
      sx={{
        display: 'flex',
        flexDirection: 'column',
        width: '900px',
      }}
    >
      <Box sx={{ display: 'flex', flexDirection: 'row' }}>
        <img
          height="45px"
          src={SupportedIntegrations[integration.service].logo}
        />
        <Box sx={{ ml: 3 }}>
          <Box display="flex" flexDirection="row">
            <Typography sx={{ fontFamily: 'Monospace' }} variant="h4">
              {integration.name}
            </Typography>
          </Box>

          {createdOnText}
          
          {serviceCard}
        </Box>
      </Box>
    </Box>
  );
```